Firebase/Database/Api/FIRDatabase.h

```diff
@@ -29,6 +29,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 FIR_SWIFT_NAME(Database)
 @interface FIRDatabase : NSObject
+  
+/**
+ * The NSObject initializer that has been marked as unavailable. Use the `database` 
+ * method instead
+ *
+ * @return An instancetype instance
```

Comment:
Actually I just saw he approved in [this comment](https://github.com/firebase/firebase-ios-sdk/issues/31#issuecomment-305355706)